fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -49,7 +49,10 @@ def load_data(self, X, y, event=None, **kwargs):
         self.index = signed.index
         self.default_objective_lambda_vec = None
 
-        # fill in the information about the basis
+        # Certain mitigation techniques require basis vectors representing the constraint space in
+        # order to generate a grid viable linear combinations. The basis vectors are stored in
+        # pos_basis and neg_basis. Including all constraints would make the vectors linearly
+        # dependent.
```

Comment:
Let's move this comment closer to where we define the basis (e.g., right above the line 58).

Also, let's use this description:
```
# The matrices pos_basis and neg_basis contain a lower-dimensional description of constraints,
# which is achieved by removing some redundant constraints.
```

Reply:
I moved it and used your description.